cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
```

Comment:
The 'specPattern' option has an incorrect value. It should be an array of glob patterns, not a string. You can fix it by wrapping the pattern in square brackets.